onnxscript/rewriter/pattern.py

```diff
@@ -954,6 +954,9 @@ def _match_node(self, pattern_node: NodePattern, node: ir.Node) -> bool:
 
         self._matched[pattern_node] = node
 
+        if len(node.inputs) != len(pattern_node.inputs):
```

Comment:
This is fine for now. We will need to revisit this to handle optional trailing inputs better.

Reply:
@xuzhenqi could you help add this as a comment in code?